tests/mozhttpd_base_test.py

```diff
@@ -37,7 +37,7 @@ def setUp(self):
         self.temp_dir = tempfile.mkdtemp()
 
     def tearDown(self):
-        mozfile.rmtree(self.temp_dir)
+        mozfile.remove(self.temp_dir)
```

Comment:
Thanks. That will remove those deprecation warnings at least. Once we converted the last unit test over to pytest, we will get rid of this file and the dependency to mozfile.

Reply:
I agree and this will increase the code coverage too.

Reply:
It shouldn't actually increase the coverage given that the lines stay the same.

Reply:
Sorry my Mistake, Yes the test coverage remains the same because the piece of code does the same thing.